draft/implementation-notes/index.html

```diff
@@ -151,42 +151,43 @@ <h2>Filesystem metadata</h2>
 
 <h2>Empty Directories</h2>
 <p>
-OCFL preserves files and their content, with directories serving as a useful organisational convention. An empty
+OCFL preserves files and their content, with directories serving as a useful organizational convention. An empty
 directory consists only of filesystem metadata and therefore, as noted above, is not amenable to direct
 preservation in OCFL. If the preservation of empty directories is considered essential then the suggested route
-is to insert a zero length file named <code>.keep</code> into the directory which will thus be preserved as
-part of the files' path.
+is to insert a zero length file named <code>.keep</code> into the directory which will ensure directories are preserved
+as part of the files' path.
 </p>
 </section><section>
 
 <h2>Data and Metadata</h2>
 <p>
 OCFL object versions are composed of series of files/bitstreams but OCFL does not make any disctinction between
-different types of files other than those reserved for OCFL functionality: the inventory and its digest file.
-It is possible, for example, to create separate data and metadata directories within each version to help
-organise material but all files are treated equally for the purpose of OCFL validation and management.
+different types of files other than those reserved for OCFL functionality: the inventory, its digest file, and
+conformance declaration files. It is possible, for example, to create separate data and metadata directories within
+each version to help organise material but all files are treated equally for the purpose of OCFL validation and
```

Comment:
`organise` to `organize`.

Reply:
/me shakes fist and curses british spelling